biogtr/data_structures.py

```diff
@@ -528,8 +543,8 @@ def __init__(
         video_id: int,
         frame_id: int,
         vid_file: str = "",
-        img_shape: ArrayLike = [0, 0, 0],
-        instances: List[Instance] = [],
+        img_shape: ArrayLike = None,
+        instances: List[Instance] = None,
```

Comment:
```suggestion
        img_shape: ArrayLike | None = None,
        instances: List[Instance] | None = None,
```

or `Optional[...]` if not py >3.10

Reply:
will add this in separate PR, probably #39